test/test_view.c

```diff
@@ -29,6 +29,7 @@ DEALINGS IN THE SOFTWARE.  */
 #include <unistd.h>
 #include <stdlib.h>
 #include <string.h>
+#include <getopt.h>
```

Comment:
Getopt.h is primarily for GNU long options.  We already use this in samtools and htslib applications (eg tabix), so this isn't an additional dependency.

Potentially we could add something like:

```
#ifdef __MSC_VER
#  include <getopt.h>
#endif
```

However it's not really worth the hassle.